hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/metrics2/util/TestSampleQuantiles.java

```diff
@@ -37,8 +37,10 @@ public class TestSampleQuantiles {
       new Quantile(0.95, 0.005), new Quantile(0.99, 0.001) };
 
   SampleQuantiles estimator;
+  Random rnd = new Random(0xDEADDEAD);
+  final static int NUM_REPEATS = 10;
 
-  @Before
+    @Before
```

Comment:
Spacing is wrong?

Reply:
In one of my commits I had unintentionally added extra space. Removed and fixed it now.